rpc_jobs/release.yml

```diff
@@ -0,0 +1,100 @@
+- job:
+    name: RE-Release
+    project-type: workflow
+    concurrent: false
+    properties:
+      - build-discarder:
+          num-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - string:
+          name: RELEASE_VERSION
+          description: |
+            The version that will be tagged by this job
+      - string:
+          name: NEXT_VERSION
+          description: |
+            The version that will be released after the next stabalisation
+            period.
+      - string:
+          name: MAINLINE
```

Comment:
nit: I think MAINLINE_BRANCH might be better here as it's a little clearer what it relates to.

Reply:
Done